test/vagrant-ci-start.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -e
+
+vagrant destroy k8s1-${K8S_VERSION} k8s2-${K8S_VERSION} --force
+vagrant up k8s1-${K8S_VERSION} k8s2-${K8S_VERSION} --provision
+
+./get-vagrant-kubeconfig.sh > vagrant-kubeconfig
+
+vagrant ssh k8s1-${K8S_VERSION} -- sudo cat /etc/kubernetes/admin.conf
```

Comment:
Is this just for debugging so it's in the logs somewhere? If so, wouldn't `cat vagrant-kubeconfig` be more correct?

Reply:
`cat vagrant-kubeconfig` will give already processed kubeconfig, this was added to show what is actually in source file. I can delete this, as it was for debugging.